{{cookiecutter.project_slug}}/clients/web/vue3/package.json

```diff
@@ -6,7 +6,7 @@
   "type": "module",
   "scripts": {
     "serve": "vite",
-    "build": "vue-tsc && vite build",
+    "build": "vite build",
```

Comment:
This would prevent your code from deploying to a review app if you have tslint issues.
Devs should always be encouraged to push experimental code to their PR. Blocking should occur when they are ready to merge to main.